src/app/tests/suites/certification/Test_TC_FLW_2_2.yaml

```diff
@@ -32,15 +32,23 @@ tests:
       command: "readAttribute"
       attribute: "MeasuredValue"
       response:
+          saveAs: ValueBeforeChange
           constraints:
               type: uint16
 
     - label: "operate on DUT to change the flow significantly"
-      disabled: true
+      PICS: MANUAL_TEMPERATURE_CHANGE
```

Comment:
What does this have to do with MANUAL_TEMPERATURE_CHANGE?

Reply:
i will update it